lib/visitors_manipulator.rb

```diff
@@ -11,7 +11,8 @@ def adjust_visitors_in_session
   end
 
   def update
-    if m = params[:next].match(/remove-(\d)/)
+    m = params[:next].match(/remove-(\d)/)
```

Comment:
Upon closer inspection of this code it appears as though it won't ever get executed. As far as I can see `params[:next]` will always just be the value of the button which is either 'Continue' or 'Add another visitor'. This will apply for both JS and non-JS page renders. Assume its intent is to remove visitors that have been added however the `build_visitors` method in the module will handle this for us as its just takes all the visitor data from the params and creates new models unless the `_destroy` value has been set in the form. I'd suggest investigating this outside of this PR as its not in scope. Test will fail if its removed, assume thats because the tests know too much about the implementation though. Also suggest refactoring it a wee bit as it does all the things.

cc @threedaymonk @SteveMarshall @kyliemulholland